### PR TITLE
aws: switch to managed node groups

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

**Comment by ddollar** [comment]
[use managed node groups for eks](https://trello.com/c/D5b9IPi7/501-use-managed-node-groups-for-eks)

File: terraform/cluster/aws/main.tf [hunk 1]
```diff
@@ -66,56 +27,35 @@ resource "aws_eks_cluster" "cluster" {
...
   ]
 
-  filename = pathexpand("~/.kube/config.aws.${var.name}")
-  content = templatefile("${path.module}/kubeconfig.tpl", {
-    ca       = aws_eks_cluster.cluster.certificate_authority.0.data
-    cluster  = aws_eks_cluster.cluster.id
-    endpoint = aws_eks_cluster.cluster.endpoint
-  })
-}
+  count = 3
```
Comment (camerondgray) [inline terraform/cluster/aws/main.tf]: I couldn't find a reference to the count argument in the docs for this. For the GCP stuff you need to you use initial_count if you enable autoscaling but that didn't seem to be a argument option for this either:  https://www.terraform.io/docs/providers/aws/r/eks_node_group.html . Am I just looking in the wrong place?

Comment (ddollar) [reply terraform/cluster/aws/main.tf]: `count` is a terraform thing. it means create n of these

Comment (camerondgray) [reply terraform/cluster/aws/main.tf]: got it. Thanks for clarifying.

**Pull request merged**